repos/backend/libs/definitions/definitions.js

```diff
@@ -24,9 +25,15 @@ const parseDefinitions = definitionFiles => {
 }
 
 const loadDefinitions = async config => {
-  const { stepsFolder } = config.paths
-  const definitionFiles = stepsFolder && await loadDefinitionsFiles(stepsFolder)
-  const definitions = await parseDefinitions(definitionFiles) || []
+  const { stepsDir, testsRoot } = config.paths
+  const pathToSteps = path.join(testsRoot, stepsDir)
+  const definitionFiles = stepsDir && await loadDefinitionsFiles(pathToSteps)
+  const herkinDefinitionFiles = await loadDefinitionsFiles(`${TEST_UTILS_PATH}/steps`)
+  const clientDefinitions = await parseDefinitions(definitionFiles) || []
+  const herkinDefinitions = await parseDefinitions(herkinDefinitionFiles) || []
```

Comment:
See my previous comment.

I don't think de-duping these is necessary because, if the client defined a step definition with the same wording, cucumber automatically throws an error explaining that there cannot be >1 registered step definition for a step. 

They would then just need to adjust their step definition's wording if they want to use their own implementation or ours.

Reply:
Ahh, I didn't know that cucumber `throws an error` when there are matching step defs. I figured it would use the last registered.
In this case no update needed